packages/@sanity/base/src/change-indicators/overlay/connectorPath.ts

```diff
@@ -0,0 +1,153 @@
+import {
+  ARROW_MARGIN_X,
+  ARROW_MARGIN_Y,
+  ARROW_SIZE,
+  ARROW_THRESHOLD,
+  CORNER_RADIUS
+} from '../constants'
+import {ConnectorLine} from './types'
+
+export function arrowPath(x: number, y: number, dir: number) {
```

Comment:
It makes me so happy to see all these things get to live in their own files :heart_eyes: :heart_eyes: